libmscore/score.h

```diff
@@ -653,6 +655,24 @@ class Score : public QObject, public ScoreElement {
       void putNote(const QPointF& pos, bool replace);
       void putNote(const Position& pos, bool replace);
       void repitchNote(const Position& pos, bool replace);
+      void cmdRealtimeAdvance();
+
+      //---------------------------------------------------------
+      //   guiSafeSleep
+      //   pause execution locally while still handling GUI events
+      //---------------------------------------------------------
+
+      void guiSafeSleep(int millis, QEventLoop::ProcessEventsFlags flags = QEventLoop::AllEvents) {
```

Comment:
@lasconic @Jojo-Schmitz where is a good place to put this function? It would be useful to have to accessible from pretty much everywhere.